src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -25,6 +25,8 @@
     Matchable,
 )
 from sqlfluff.core.dialects import load_raw_dialect
+from sqlfluff.core.parser.lexer import StringLexer
+from sqlfluff.core.parser.segments.raw import KeywordSegment
```

Comment:
We import these from `sqlfluff.core.parser` to avoid lots of these imports.

Reply:
VSCode did it! Yes, I saw you mention this to someone else.
The ordering of the parse imports seems haphazard. OK, if I alphabetize?

Reply:
Not only am I OK if you alphabetize - I'd prefer it! Was actually looking at another MySQL issue and was going to alphabetize it there myself. But turns out I can't fix that. So please do in this PR!